form.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>

```

Comment:
Here you did the same mistake, this file should be in different branch, not here